lib/jpeg.xml

```diff
@@ -0,0 +1,149 @@
+  <name>Jpeg</name>
+  <summary xml:lang="en">Jpeg: library and tools for JPEG images</summary>
+  <description xml:lang="en">This package contains C software to implement JPEG image compression and decompression. JPEG is a standardized compression method for full-color and gray-scale images. JPEG is intended for &quot;real-world&quot; scenes; cartoons and other non-realistic images are not its strong suit. JPEG is lossy, meaning that the output image is not identical to the input image. The user can trade off output image quality against compressed file size by adjusting a compression parameter. 
+
+The distributed programs provide conversion between JPEG &quot;JFIF&quot; format and image files in PBMPLUS PPM/PGM, GIF, BMP, and Targa file formats. The core compression and decompression library can easily be reused in other programs, such as image viewers. 
+
+Includes EXIF patches from: 
+http://bugzilla.redhat.com/bugzilla/show_bug.cgi?id=106060 http://sylvana.net/jpegcrop/exifpatch.html 
+</description>
+  <icon href="https://raw.githubusercontent.com/0install/repo.roscidus.com/master/lib/jpeg.ico" type="image/vnd.microsoft.icon"/>
+  <icon href="https://raw.githubusercontent.com/0install/repo.roscidus.com/master/lib/jpeg.ico" type="image/png"/>
+  <category>Graphics</category>
+  <homepage>https://web.archive.org/web/20171208102837/http://gnuwin32.sourceforge.net/packages/jpeg.htm</homepage>
+  <needs-terminal/>
+  <group arch="Windows-i486" license="BSD License (original)" version="6-4">
+    <command name="jpegexiforient" path="bin/jpegexiforient.exe"/>
+    <command name="jpegtran" path="bin/jpegtran.exe"/>
+    <command name="run" path="bin/rdjpgcom.exe" />
+    <command name="wrjpgcom" path="bin/wrjpgcom.exe"/>
+    <command name="exifautotran" path="bin/exifautotran">
+      <executable-in-path command="jpegexiforient" name="jpegexiforient"/>
+      <executable-in-path command="jpegtran" name="jpegtran"/>
+      <requires interface="http://repo.roscidus.com/utils/coreutils">
+        <executable-in-path command="rm" name="rm"/>
+        <executable-in-path command="mv" name="mv"/>
+      </requires>
+      <runner interface="http://repo.roscidus.com/utils/bash"/>
+    </command>
+    <implementation id="sha1new=5a950b76f802d59d7a031c601e724c250125818e" version="6-4-3">
+      <environment insert="bin" name="PATH"/>
+      <!--command name="cjpeg" path="bin/cjpeg.exe">
+        <requires interface="http://repo.roscidus.com/lib/urt.xml" version="3.1.2.1-2-1">
+          <environment insert="bin" name="PATH"/>
+        </requires>
+      </command>
+      <command name="djpeg" path="bin/djpeg.exe">
+        <requires interface="http://repo.roscidus.com/lib/urt.xml" version="3.1.2.1-2-1">
+          <environment insert="bin" name="PATH"/>
+        </requires>
+      </command-->
```

Comment:
Why are these `<command>`s commented out?

Reply:
jpegs dependency urt is commented out to be restored if/when it is available